flight/mcu/lib/teensyShared/rk4.cpp

```diff
@@ -0,0 +1,90 @@
+#include "rk4.h"
```

Comment:
blurb.

Reply:
@parthshrotri follow the doxygen format for a file in something like dataLog.cpp. nothing too long, just a little explanation of what's in the file.